release/RELEASE.md

```diff
@@ -285,7 +285,7 @@ patches we backport to the 1.0 release branch).
 
    > Manual tests passed on Windows.
 
-1. If the release is bad, delete the release from [the releases page]. Mention
+1. If the release is bad, delete the release from the [releases page]. Mention
```

Comment:
Please make this as a separate PR. I'm actually not sure how CI will react to a release PR that is not a single commit. 🙁

Reply:
Well, I had to add the annoying empty changelog markers...

Reply:
You can amend the previous commit to add the changelog markers. The release commit should only contain the change to the `LATEST` file.

Reply:
From [the release detection step](https://dev.azure.com/digitalasset/daml/_build/results?buildId=64226&view=logs&j=e74c1ec1-54ae-5943-cedf-e785163662c7&t=68519945-eab9-5f94-7ad3-9048857f8c49):
```
Setting 'is_release' to 'false'
```

Reply:
empty commits also work for adding changelog entries

Reply:
I reread the [code for that step](https://github.com/digital-asset/daml/blob/master/azure-pipelines.yml#L343-L381). You can have as many commits as you want, but the total effect of your PR must be a single line of change to the `LATEST` file.